### ISSUE TITLE
Test failure with geos 3.9.0

### ISSUE THREAD

**Issue by hjaekel** [Toblerity/Shapely#1079]
## Expected behavior and actual behavior.

The test `SvgTestCase.test_collection` fails for Shapely 1.7.1 with geos 3.9.0. 

```
======================================================================
FAIL: test_collection (tests.test_svg.SvgTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/builder/package/community/py3-shapely/src/Shapely-1.7.1/tests/test_svg.py", line 176, in test_collection
    self.assertSVG(
  File "/home/builder/package/community/py3-shapely/src/Shapely-1.7.1/tests/test_svg.py", line 41, in assertSVG
    self.assertEqual(svg_elem, expected)
AssertionError: '<g><polyline fill="none" stroke="#66cc99" str[158 chars]</g>' != '<g><circle cx="7.0" cy="3.0" r="3.0" stroke="[158 chars]</g>'
- <g><polyline fill="none" stroke="#66cc99" stroke-width="2.0" points="4.0,2.0 8.0,4.0" opacity="0.8" /><circle cx="7.0" cy="3.0" r="3.0" stroke="#555555" stroke-width="1.0" fill="#66cc99" opacity="0.6" /></g>
+ <g><circle cx="7.0" cy="3.0" r="3.0" stroke="#555555" stroke-width="1.0" fill="#66cc99" opacity="0.6" /><polyline fill="none" stroke="#66cc99" stroke-width="2.0" points="4.0,2.0 8.0,4.0" opacity="0.8" /></g>
```

So just the order of the polyline and the circle in the SVG changed.

## Steps to reproduce the problem.

Use `python3 setup.py build`

## Operating system

Alpine Linux Edge

## Shapely version and provenance

Shapely 1.7.1

**Comment by jorisvandenbossche** [comment]
Yes, the coordinate pair order has changed for several operations in GEOS 3.9.0, and hence causing some failures in existing tests. 
This has been fixed in the meantime: https://github.com/Toblerity/Shapely/pull/1042 (note that it is only the test itself that failed due to a slightly changed expected result, there is no actualy runtime failure when using Shapely 1.7 with GEOS 3.9.0)

**Comment by sgillies** [comment]
@jorisvandenbossche thanks for the explanation! @hjaekel there will be a fix when 1.8.0 is released. That's a little ways out yet.
